planetary-social/nos
# Tap to Refresh in empty profiles

## Issues covered
#1068

## Description

Adds a Tap to Refresh button and refreshes the Profile when tapped. I used the same functions we use when pulling down to refresh. Also, the ticket specifically requires that the button shows the same loading indicator we show when the user pulls down to refresh.

## How to test
1. Navigate to your profile
2. Delete some posts if they are just a few (otherwise, you'll need to create a new profile)
3. Write a new post
4. See how that new post appears after tapping on "Tap to refresh"

## Screenshots/Video

![Simulator Screenshot - iPhone 15 - 2024-07-05 at 17 22 33](https://github.com/planetary-social/nos/assets/1703242/7d0b8a2e-71a1-4f3b-afd0-6e68952c000c)
